### PR TITLE
Adds a power knot to xenoarch

### PR DESCRIPTION
**Pull request by LoreCoder**
Used for powering emitters, which are needed for certain anomalies. (see below)
image

### CODE REVIEW

File: html/changelogs/researchpowerknot.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Adds a power knot to Xenoarcheology for powering emitters."
```
Comment (listerla) [inline html/changelogs/researchpowerknot.yml:41]:
```suggestion
  - rscadd: "Adds a power knot to Xenoarchaeology for powering emitters."
```

**Comment by LoreCoder** [comment]
!review

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
huge diff, see if you can fix it somehow.

**Comment by LoreCoder** [comment]
closing to fix diff issues

**Pull request closed without merging**